### DISCUSSION TITLE
[Question]Does what I did have meaning?我做的这个事有意义吗？

### DISCUSSION THREAD

**Discussion by LegendaryPistachio** [flet-dev/flet#4290] [Show and tell]
### Question

Does what I did have meaning?”
我做的这个事有意义吗？

About
Developing a Repository for Custom Components, Custom UI Templates, and Custom Animation Templates using the Flet Framework.

使用Flet框架开发自定义组件、自定义用户UI界面模板和自定义动画模板的存储库

![微信图片_20241102112514](https://github.com/user-attachments/assets/86ad55e5-19eb-4d53-9ee1-0cbefc876279)
![微信图片_20241102112533](https://github.com/user-attachments/assets/47a51398-c2d3-4b72-b879-416f78780fe9)

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ap4499** [comment]
Apologies for the semantics "Does what I did have meaning?”

Is your intended meaning "Is this useful to people?" (sorry, I'm European and only speak English/Spanish!).

But yes - I think it is useful. There isn't always the development budget to create advanced bespoke animations, and pooling together a resource of projects that have neat animations or reusable components would be really cool.

**Reply by LegendaryPistachio** [reply]
very good!

**Reply by SriniG-333** [reply]
Yes !
